scripts/publish.sh

```diff
@@ -118,6 +118,20 @@ publish_to_docker_hub() {
 
 }
 
+publish_to_public_ecr() {
+	aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/aws-observability
+
+	for arch in "${ARCHITECTURES[@]}"
+	do
+		docker tag ${1}:"$arch" public.ecr.aws/aws-observability/aws-for-fluent-bit:"$arch"-${AWS_FOR_FLUENT_BIT_VERSION}
+		docker push public.ecr.aws/aws-observability/aws-for-fluent-bit:"$arch"-${AWS_FOR_FLUENT_BIT_VERSION}
+	done
+
+	create_manifest_list public.ecr.aws/aws-observability/aws-for-fluent-bit ${AWS_FOR_FLUENT_BIT_VERSION} ${AWS_FOR_FLUENT_BIT_VERSION}
+	aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/aws-observability
```

Comment:
Do we need this again? We got the login credentials in line 122.

Reply:
yes we need it. For some reason, we need to got it every time before we want to `create manifest list`. If we don't add this line, we would have an error.

Reply:
nit: Hmm, interesting. [Here](https://github.com/aws/aws-for-fluent-bit/pull/124/files#diff-c63759f58a022cf694046a961470c6e28c042d8f221646bc5ddbb5fa959dda75R255) they did the same but for regional ECR I guess. Do we have the error log?

Reply:
yes we have.
<img width="1323" alt="Screen Shot 2020-12-18 at 1 55 43 PM" src="https://user-images.githubusercontent.com/71519901/102932595-6dcd1d80-4455-11eb-9f98-be45c7ba8f69.png">

Reply:
You can see here. I didn't log in again in my previous code and it succeeds with `tag: 2.10.0` but failed with `tag:latest`. The error info is `denied: Not Authorized`.